app/components/story-list/template.hbs

```diff
@@ -1,7 +1,8 @@
-<ul class="list list--noborder">
+<ul class="list list--borderbottom">
```

Comment:
@marineb is this going to update all the other show pages. I think this needs to be conditionally applied based on the value of `flipTease`, right?